hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/WorkingStateAccessor.java

```diff
@@ -18,38 +18,38 @@
 
 import static java.util.Objects.requireNonNull;
 
-import com.swirlds.state.MerkleState;
+import com.swirlds.state.State;
 import edu.umd.cs.findbugs.annotations.Nullable;
 import javax.inject.Inject;
 import javax.inject.Singleton;
 
 /**
- * A singleton class that provides access to the working {@link MerkleState}.
+ * A singleton class that provides access to the working {@link State}.
  */
 @Singleton
 public class WorkingStateAccessor {
-    private MerkleState merkleState = null;
+    private State state = null;
 
     @Inject
     public WorkingStateAccessor() {
         // Default constructor
     }
 
     /**
-     * Returns the working {@link MerkleState}.
-     * @return the working {@link MerkleState}.
+     * Returns the working {@link State}.
+     * @return the working {@link State}.
      */
     @Nullable
-    public MerkleState getMerkleState() {
-        return merkleState;
+    public State getMerkleState() {
```

Comment:
Should this be called getState / setState accordingly?

Reply:
Yes, it should. I'll fix it

Reply:
Done